ozaria/site/templates/play/tome/spell_palette_entry_popover.jade

```diff
@@ -150,49 +37,88 @@ mixin argumentEntry(arg)
             span(dir="rtl")= ": "
           code(dir="ltr")= arg.default
 
-if doc.args && doc.args.length
-  - var hasOptionalArguments = _.any(doc.args, function(arg){ return arg.optional })
-  - var hasRequiredArguments = _.any(doc.args, function(arg){ return !arg.optional })
-  if hasRequiredArguments
-    p.args(dir="auto")
-      strong
-        span(data-i18n="skill_docs.required_parameters") Required Parameters
-        span.spr :
-    for arg in doc.args
-      unless arg.optional
-        +argumentEntry(arg)
-  if hasOptionalArguments
-    p.args(dir="auto")
-      strong
-        span(data-i18n="skill_docs.optional_parameters") Optional Parameters
-        span.spr :
+div.popover-header.closeBtn(role="button" dir="ltr")
+  span.prop-name= doc.shortName
+  if doc.translatedShortName
+    span.translated-name (
+      =doc.translatedShortName
+      | )
```

Comment:
Is this to make an intentional space here? I wonder if that is better than the character code I've been using sometimes.

Reply:
This is just to add the closing braces after the translatedShortName without a space (for example: https://stackoverflow.com/questions/27247407/how-to-put-span-in-jade)